### ISSUE TITLE
Minor translation issues

### ISSUE THREAD

**Issue by stemabu** [robert7/nixnote2#76]
Hi Robert,
great that you continuing this project! I updated the German translation to catch up with the recent changes. As I'm not familiar with the usual project process, I hope it is ok that I filed it as a pull request. If not, please let me know how to do it.

In doing the translation I noticed the following minor issues:
1) There is no string in the translation file for "Language *" in the locale tab of the preferences window.
2) There are no strings in the translation files in the editor window for the color names at the buttons of text color and text background color. There are strings for color names, but the translations appear at the note's background color in the context menu only. Could these maybe also be used for the other color selections?
3) On the "Data log and location info" window I had difficulties to put every German translation in one line. Would it maybe possible to widen the window a bit? Furthermore, with long file paths I got line breaks in the paths. So I thought if there could be a line break after the colon so that the file path gets always a new line.

Best regards,
Martin

**Comment by robert7** [comment]
thx! I'll check the pull request

**Comment by robert7** [comment]
I merged the pull request (so far only to the development version - "2.1.3-a") and fixed the mentioned issues.
You can further update the german translation - I added few german color names, but not all.
I also synced the translation files with source, so there are few new texts (you can find them by `translation type="unfinished"`)

![screen_20190105_01](https://user-images.githubusercontent.com/1135218/50724001-1b6a8900-10e6-11e9-8b46-01d8b4b3f87c.png)

![screen_20190105_02](https://user-images.githubusercontent.com/1135218/50724002-1f96a680-10e6-11e9-8389-0540e8acdc7f.png)

![screen_20190105_04](https://user-images.githubusercontent.com/1135218/50724003-258c8780-10e6-11e9-8331-2b381f73bf0d.png)

I hope I didn't screw the color menu logic. Pls do few tests, whenever all works fine with local colors.
You can also put your name in the translation line, if you want (but its optional of course).

& vielen Dank & schöne Grüsse :smile: 
Robert

**Comment by stemabu** [comment]
Danke, Robert! :-)

I did some testing on the colors and more translations. I doing this, I came across these minor issues:
1) Gray and Darkgray are exchanged in the color menu. So the Gray label brings dark gray color, also the square on its left shows dark gray (see also in your screenshot above). This seems to be an older issue. As it affects all languages, I did not correct it with the translation, but hope you can fix it. I would suggest to exchange the labels of gray and darkgray and then exchange the order of the two to bring back gray above dark gray.
2) In editorbuttonbar.cpp there is a typo: "In**t**ent" instead of "Indent". I did not know about this menu before I searched where the translation belongs to and was happy to find further editing buttons there. :-) Maybe you could consider a button to call up the menu as I guess I'm not the only one who do not know about it.
3) In nwebview.cpp I found "Tidy HTML" as well as "Simplify Formatting". But I could not find "Tidy HTML" in the context menu were I found "Simplify Formatting". Concerning the translation, I was thinking what the functional difference between the two is to find a proper translation. Can you help me on this?
4) I am not sure how to proper translate "exit". I think the German word "Ausgang", which would be the direct translation, is not helpful for any user. But I could not find a technical term, which would fit – maybe this is because I'm not a programmer. I thought to use "Skripte (Exits)" – i.e. "Scripts (exits)" – as translation for "exits", what I think would describe it, but I'm not sure if it is ok if the translation differs so far from the original. What do you think?
5) I came across these English words, of which I could not find a translation entry:
a) "yes" and "no" at the verify delete window
b) "cancel", "save", "all files" and "nixnote export" in the file windows of database backup and exports

Schöne Grüße
von Martin

**Comment by robert7** [comment]
Hi,
so I did basically all points.
1, should now work now
2. fixed. Additionally I added a text hint, which is displayed on "hover" on "font type/size" selection. There is a chance every one will get it displayed sometime. And there is "Getting started" wiki page, where this topic (and other "tips") are covered.
3, "tidy html" menu option was hidden (was only used for testing) and I removed it now. Tidy is applied automatically before note is saved.
4, "Ausgang" would be funny :) but wouldn't help.. The wording "Exits" was selected by original author.
See: https://sourceforge.net/p/nevernote/mailman/message/35960951/
But it seems to cause confusion so I changed the name in English to "Scripting (Exits)" and for german "Skripte (Exits)" then seems to be fine. Basically its "scripts" which can be used as "Hooks" ... "Exits" is a bit unusual wording.
5, verify delete window yes/no should be now translatable, also the at the "note restore"
the file dialogs are now not translatable; I would expect when you run the system in german, that they display in german, but they doesn't; I may look at it later
I also change the working for the "database backup" and "database restore" as internally they are equal to "export notes" - they just "export all notes"

So few more text needs translation, but overall the german version seems to be fine.

The changes are available from version "2.1.3-b" (currently available in development version binaries).

lg
Robert
